src/components/networks/networkOverviewTab.jsx

```diff
@@ -134,12 +158,40 @@ export const NetworkOverviewTab = ({ network }) => {
                         <DescriptionListDescription id={`${idPrefix}-ipv6-prefix`}> {ip[1].prefix} </DescriptionListDescription>
                     </DescriptionListGroup> }
 
-                    { ip[1].dhcp.range.start && <DescriptionListGroup>
-                        <DescriptionListTerm> {_("DHCP range")} </DescriptionListTerm>
-                        <DescriptionListDescription id={`${idPrefix}-ipv6-dhcp-range`}> {ip[1].dhcp.range.start + " - " + ip[1].dhcp.range.end} </DescriptionListDescription>
-                    </DescriptionListGroup> }
+                    <Text component={TextVariants.h5}>
+                        {_("DHCP Settings")}
+                    </Text>
 
-                    { ip[1].dhcp.hosts.map((host, index) => DHCPHost(host, index, ip[1].family, idPrefix))}
+                    { ip[1].dhcp.range.start &&
+                    <DescriptionListGroup>
+                        <DescriptionListTerm> {_("Range")} </DescriptionListTerm>
+                        <DescriptionListDescription id={`${idPrefix}-ipv6-dhcp-range`}>
+                            {ip[1].dhcp.range.start + " - " + ip[1].dhcp.range.end}
+                        </DescriptionListDescription>
+                    </DescriptionListGroup>}
+                    <DescriptionListGroup>
+                        <DescriptionListTerm>
+                            <Flex>
+                                <FlexItem>{_("Static host entries")}</FlexItem>
+                                <NetworkAddStaticHostEntries idPrefix={idPrefix} network={network} parentIndex="1" />
+                            </Flex>
+                        </DescriptionListTerm>
+                        <DescriptionListDescription id={`${idPrefix}-ipv6-dhcp-static`}>
+                            <List isPlain>
+                                { ip[1].dhcp.hosts.length
+                                    ? ip[1].dhcp.hosts.map((host, index) => DHCPHost(host, index, ip[1].family, idPrefix))
+                                    : <ListItem>{_("none")}</ListItem>
+                                }
+                            </List>
+                        </DescriptionListDescription>
+                    </DescriptionListGroup>
+
+                    <List isPlain>
+                        { ip[1].dhcp.hosts.length
+                            ? ip[1].dhcp.hosts.map((host, index) => DHCPHost(host, index, ip[1].family, idPrefix))
+                            : <ListItem>{_("none")}</ListItem>
+                        }
+                    </List>
```

Comment:
I think there is, by accident, the same code twice, which causes to render "none" twice in IPv6's Static host entries section
![Screenshot from 2021-09-30 01-03-37](https://user-images.githubusercontent.com/42733240/135360242-9182787d-dd8a-4bb1-b193-1f0ca5ec3fe6.png)

Reply:
Fixed.